src/app.py

```diff
@@ -76,262 +17,5 @@ async def lifespan(app: FastAPI):
-        background_tasks.add_task(spawn_process_for_new_agent, object)
-    logger.debug(f"Object has been successfully added to scene")
-    return {"serialized_object": object.serialize()}
-
-
-@app.post("/tick")
-async def tick():
-    logger.debug(f"Starting to tick scene: {state.tick_number}")
-    actions = await fetch_actions()
-    actions.extend([Entity.deserialize(pending_action) for pending_action in state.scene.pending_actions])
-    state.scene.pending_actions.clear()
-    applied_actions = apply_actions(actions)
-    state.transitions.append(applied_actions)
-    state.tick_number += 1
-    await send_scene()
-    logger.debug(f"Scene has been successfully ticked: {state.tick_number}")
-    return {"tick_number": state.tick_number}
-
-
-@app.post("/clear")
-async def clear():
-    logger.debug(f"Starting to clear scene")
-    state.scene = Scene()
-    state.processes = {}
-    state.transitions = [[]]
-    state.tick_number = 0
-    logger.debug(f"Scene has been successfully cleared")
-
-
-@app.post("/populate")
-async def populate():
-    logger.debug(f"Starting to populate scene")
-    cancel_global_tasks()
-    await clear()
-    opensimplex.seed(1234)
-    id = 0
-    for (x, y) in itertools.product(range(int(config["REQUESTS"]["MAP_SIZE_X"])), range(int(config["REQUESTS"]["MAP_SIZE_Y"]))):
-        state.scene.add_entity(Object(id=id, name="Grass",
-                                      position=Vector(x, y), tags=['walkable']))
-        id += 1
-        if opensimplex.noise2(x, y) > .3:
-            entity_name = "Tree"
-            if id % 2 == 0:
-                entity_name = "Rock"
-            state.scene.add_entity(
-                Object(id=id, name=entity_name, position=Vector(x, y)))
-            id += 1
-    task = asyncio.create_task(create_tick())
-    tick_trigger_background_task.append(task)
-    logger.debug(f"Scene has been successfully populated")
-    return {"id": id}
-
-
-@app.post("/generate_scene")
-async def generate_scene():
-    logger.debug(f"Starting to generate scene")
-    import os
-
-    url = os.getenv('LYNX_SCENE_GENERATOR_URL')
-
-    if url == None:
-        url = 'https://scene-generator.kubernetes.blazej-smorawski.com/get_scene'
-
-    await clear()
-    response = await post(url, payload={"seed": "test", "width": int(config["REQUESTS"]["MAP_SIZE_X"]), "height": int(config["REQUESTS"]["MAP_SIZE_y"])})
-    state.scene = Scene.deserialize(response.text)
-    logger.debug(f"Scene has been successfully generated")
-
-
-# Teardown is necessary to close all subprocesses
-# I tired using FastAPI `lifespan` but it might not work
-# with apps that are not top-level
-
-
-@app.post('/teardown')
-async def teardown():
-    logger.debug(f"Teardown")
-    close_processes()
-    return {"status": "done"}
-
-
-@app.get('/health')
-async def health():
-    logger.debug(f"Health check")
-    return {"state": "working"}
+app.include_router(scene_router.router)
+app.include_router(server_router.router)
```

Comment:
`include_router` is used to group similair endpoints together right?

Reply:
yep, you can create mutltiple routers, and include them here